plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml

```diff
@@ -260,18 +280,18 @@
           siteUri="https://cloud.google.com/storage/docs/reference/libraries#client-libraries-install-java" >
       <libraryDependency id="googlecloudcore" />
       <libraryFile
-            javadocUri="https://googlecloudplatform.github.io/google-cloud-java/0.10.0/apidocs/">
+            javadocUri="https://googlecloudplatform.github.io/google-cloud-java/0.20.1/apidocs/">
         <mavenCoordinates
               artifactId="google-cloud-storage"
               groupId="com.google.cloud"
-              version="1.0.0" />
+              version="1.2.1" />
       </libraryFile>
       <libraryFile
             javadocUri="https://developers.google.com/resources/api-libraries/documentation/storage/v1/java/latest/index.html">
         <mavenCoordinates
               artifactId="google-api-services-storage"
               groupId="com.google.apis"
-              version="v1-rev102-1.22.0" />
+              version="v1-rev100-1.22.0" />
```

Comment:
Turns out we weren't using the correct version in the first place:  [`google-cloud-storage` 1.0.0 references `v1-rev100-1.22.0` in too](https://repo1.maven.org/maven2/com/google/cloud/google-cloud-storage/1.0.0/google-cloud-storage-1.0.0.pom).